Apps/Playground/android/build.gradle

```diff
@@ -3,16 +3,17 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 16
+        minSdkVersion = 24
```

Comment:
Is this required? I guess maybe it's for ARCore, but changing the minimum from 16 to 24 means about 30% of Android devices in the wild won't be able to use this at all (if it works the way I think it works).

https://developer.android.com/about/dashboards/

Reply:
I ran into something that required me to bump it to 21, and then something else that went to 24. I think it was the use of CompletionStage<T> or something, but I can't remember for sure. I know I did this for a reason though, so my vote would be to wait until we have evidence that consumers of the lib want it, and then we can re-evaluate.